hotsos/plugin_extensions/openstack/agent/exceptions.py

```diff
@@ -290,7 +293,7 @@ def summary_agent_exceptions(self):
 
         return None
 
-    @summary_entry('agent-warnings', 201)
+    @summary_entry('agent-warnings', get_min_available_entry_index() + 201)
```

Comment:
The previous index was 201 whereas the new one is 207. Does it matter here?

Reply:
no its arbitrary here, as long as it is after the others its ok